docs/app/pages/css/css-sections/icons/ux-icons/icon-preview/icon-preview.component.html

```diff
@@ -11,6 +11,7 @@
 
     <ux-icon
```

Comment:
Should the text be vertically centered with the icon?

![image](https://user-images.githubusercontent.com/20795331/62872193-d9a55f00-bd14-11e9-97a6-a47940e16d6d.png)